### ISSUE TITLE
TypeError Mezatsong\SwaggerDocs\Parameters\BodyParametersGenerator::addToProperties(): Argument #1 ($properties) must be of type array, null given

### ISSUE THREAD

**Issue by Mezatsong** [Mezatsong/laravel-swagger-docs#5]
I have an issue when I put an array in validation, specialy when the name of the rule end with *

![image](https://user-images.githubusercontent.com/42788152/233395386-f2c8d6cd-2a6d-44d8-aea4-86bfda1ecdad.png)

The FormRequest:
```php
public function rules()
    {   
        return [
            'label' => 'required|string',
            'type' => 'required|string|in:' . implode(',', TestType::allCaseInStr()),
            'simulateur_id' => 'required|exists:simulateurs,id',
            'time' => 'required|numeric|min:1', // en seconde
            'lang' => 'required|string|size:2',
            'sub_title' => 'nullable|string',

            'questions' => 'required|array|min:1',
            'questions.*.title' => 'required|string',
            'questions.*.images' => 'present|array',
            'questions.*.images.*' => 'required|base64image',
            'questions.*.content' => 'nullable|string',
            'questions.*.explanation' => 'nullable|string',
            'questions.*.explanation_images' => 'present|array',
            'questions.*.explanation_images.*' => 'required|base64image',
            'questions.*.max_responses' => 'required|numeric|min:0',

            'questions.*.answers' => 'required|array|min:1',
            'questions.*.answers.*' => 'required',
            'questions.*.answers.*.content' => 'required|string',

            'questions.*.answers.*.is_correct' => 'required|boolean',
            'questions.*.answers.*.images' => 'present|array',
            'questions.*.answers.*.images.*' => 'required|base64image',
        ];
       ```

The Error:
```

   TypeError 

  Mezatsong\SwaggerDocs\Parameters\BodyParametersGenerator::addToProperties(): Argument #1 ($properties) must be of type array, null given, called in C:\User
s\B_Soft\Projects\Free-lance\adaa\codes\api.adaacertif.com\vendor\mezatsong\laravel-swagger-docs\src\Parameters\BodyParametersGenerator.php on line 122      

     92▕         if (\count($nameTokens) === 0) {
     93▕             return;
     94▕         }
     95▕

  1   C:\Users\B_Soft\Projects\Free-lance\adaa\codes\api.adaacertif.com\vendor\mezatsong\laravel-swagger-docs\src\Parameters\BodyParametersGenerator.php:122 
      Mezatsong\SwaggerDocs\Parameters\BodyParametersGenerator::addToProperties()

  2   C:\Users\B_Soft\Projects\Free-lance\adaa\codes\api.adaacertif.com\vendor\mezatsong\laravel-swagger-docs\src\Parameters\BodyParametersGenerator.php:120 
      Mezatsong\SwaggerDocs\Parameters\BodyParametersGenerator::addToProperties()
      ```

**Comment by evolvoltd** [comment]
Hi, quick fixes-patches for this issue and issue 6 https://github.com/Mezatsong/laravel-swagger-docs/pull/8

**Comment by Mezatsong** [comment]
> Hi, quick fixes-patches for this issue and issue 6 #8

Thank you
